.github/workflows/add-prs-and-issues-to-project.yml

```
@@ -0,0 +1,21 @@
+name: Adds all new issues and PRs to the Transition Monitor project
+
+on:
+  issues:
+    types:
+      - opened
+      - transferred
+  pull_request:
+    branches: [main]
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.4.0
+        with:
+          project-url: https://github.com/orgs/RMI-PACTA/projects/8
+          github-token: ${{ secrets.CJ_ADD_TO_PROJECT_PAT }}
```

```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

So we don't rely on any one person's PATs? 

https://docs.github.com/en/actions/security-guides/automatic-token-authentication

---

awesome! assuming it works... does it have "project" access?

---

Actually, I think it needs to be a repo secret or an org secret, and the `GITHUB_TOKEN` is a special auto secret that only has access to the repo it's used in? This needs access to the org project that is out of scope for this repo.

Maybe `REPO_PAT` which is added to some of these repos as an org secret? Does that have access to projects?

---

it looks like the automatic token has access to the `repository-projects` scope. Not sure if that's what we need or not though.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

---

🤔 Hmm. you're probably right. I don't think that I added the projects scope to `REPO_PAT`, but I could. That would probably keep things a bit cleaner.

---

I see there are a lot of PRs open with the same general gist. I suppose they all need to wait for this conversation to get resolved?

---

yeah, I think so

---

Any update on this?

---

Still need to create an org level PAT for it @AlexAxthelm

---

```suggestion
          github-token: ${{ secrets.PAT_ADD_ISSUES_TO_PROJECT }}
```

I've created the @PACTA-bot user who created the PAT_ADD_ISSUES_TO_PROJECT PAT with repo and project scopes.